src/js/components/SchemaForm.js

```diff
@@ -118,9 +118,15 @@ class SchemaForm extends React.Component {
   }
 
   getNewDefinition() {
-    return SchemaUtil.schemaToMultipleDefinition(
+    let definition = SchemaUtil.schemaToMultipleDefinition(
       this.props.schema, this.getSubHeader, this.getLabel
     );
+
+    if (this.props.model) {
+      SchemaFormUtil.mergeModelIntoDefinition(this.props.model, definition);
```

Comment:
1. This function doesn't exist yet?
2. And is it supposed to modify `definition`? Not sure that's the best path to go down. maybe just return a new `definition`?